mismi-s3/src/Mismi/S3/Data/Lens.hs

```diff
@@ -0,0 +1,26 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Mismi.S3.Data.Lens (
```

Comment:
I feel like this is an unrelated changed and should probably be a separate PR, mainly because I think this is a larger discussion. We are currently only using records afaik (at least in mismi), should we add a second way to interact with data types via lenses or choose a standard method? /discussion to be had